packages/schematics/README.md

```diff
@@ -17,7 +17,7 @@ and configured automatically for you.
 | ----------------------------------- | ------------------------------------------------------------ |
 | `ng g @co-it/schematics:commitlint` | Configure [commitlint] and set up commit hook using [husky]. |
```

Comment:
**Rephrase Proposal**
Set up [commitlint] and optionally configure a commit hook using [husky].